docs/platform/server/api.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Falcon Server API
+---
+
+> This section provides an API description for both packages - `@deity/falcon-server`
+> and `@deity/falcon-server-env`
+
+## Dependency Inversion
+
+In order to bring more flexibility in dependencies instance lifetime management, Falcon Server internally implements Dependency Inversion pattern via [Iversify JS](http://inversify.io/) framework.
+It means that Falcon Server internals, including extension components (Api DataSources, Extensions end Endpoints) are registered in Inversify container. Even they are setted up via configuration files.
```

Comment:
we could add more info about some internal things that we did, like getting ApiDataSources by using registered keywords, like `Config`, `EventEmitter2`, containers (Api, Endpoint, Extension), entries by type - `ApiDataSource`, `{name}DataSource`, components and so on. These things are highly useful and important to know.